apps/src/code-studio/components/ShareAllowedDialog.jsx

```diff
@@ -438,6 +483,30 @@ class ShareAllowedDialog extends React.Component {
                     </span>
                   </div>
                 )}
+                {showPublishInfo &&
+                  !isLoadingAccountAndProjectAge &&
+                  !isAccountOldEnoughToPublish && (
+                    <div style={{clear: 'both', marginTop: 10}}>
```

Comment:
nit: this div and the one below are the same except the text, can we factor into a helper?

Reply:
👍 Pulled out these two as well as other similar warnings into a helper